flips/20201125-bip-44-multi-account.md

```diff
@@ -9,188 +9,279 @@
-key derivation paths that support multiple coins and multiple accounts per coin.
+BIP 44 defines an implementation of [Bitcoin Improvement Proposal
+32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
+that standardizes a format for hierarchical deterministic (HD) key derivation
+paths that support multiple coins and multiple accounts per coin.
 
 The BIP 44 path format is as follows:
 
 ```
 m / purpose' / coin_type' / account' / change / address_index
 ```
 
-When using this format to construct or discover a Flow user wallet, 
-the value(s) of each path parameter should follow the specifications below.
+When using this format to construct or discover a Flow user wallet, the value(s)
+of each path parameter should follow the specifications below.
 
 #### Purpose
 
-Purpose is always the constant 44' (0x8000002C), 
-as described in [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose).
+Purpose is always the constant 44' (0x8000002C), as described in [BIP
+44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose).
 
 #### Coin Type
 
-Coin type is always the constant 539' (0x8000021b), 
-the unique identifier for the Flow protocol, as defined in 
-[SatoshiLabs Improvement Proposal 44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (SLIP 44).
+Coin type is always the constant 539' (0x8000021b), the unique identifier for
+the Flow protocol, as defined in [SatoshiLabs Improvement Proposal
+44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (SLIP 44).
 
 #### Account
 
-Account is the index of a distinct account address owned by the wallet user. 
-No two account indices should be used to generate child public keys that are assigned 
-to the same Flow account address. Furthermore, a single account index should not
-be used to generate keys for more than one account address.
+Account is the index of a distinct account address owned by the wallet user. No
+two account indices should be used to generate child public keys that are
+assigned to the same Flow account address. Furthermore, a single account index
+should not be used to generate keys for more than one account address.
+
+The Account Index to use when generating a key pair to be added on a new account
+is the smallest Account Index of which has not yet been used to generate keys 
+set on any account.
 
 The motivation for these accounts is the same set forth in BIP 44:
 
-> Users can use these accounts to organize the funds in the same fashion as bank accounts; for donation purposes (where all addresses are considered public), for saving purposes, for common expenses etc.
+> Users can use these accounts to organize the funds in the same fashion as bank
+> accounts; for donation purposes (where all addresses are considered public),
+> for saving purposes, for common expenses etc.
+
+Unlike BIP 44, wallets should not prevent the creation of a new account if any
```

Comment:
I might rephrase this for clarity as follows:

Key difference: BIP 44 requires that wallets check the accounts associated with previous account indices. If a previous account does not have a transaction history, creation of a new account is prevented. This is expressly not a requirement in this specification (FLIP-200).